app/javascript/packs/document-capture.tsx

```diff
@@ -155,6 +156,15 @@ const App = composeComponents(
       maxSubmissionAttemptsBeforeNativeCamera: Number(maxSubmissionAttemptsBeforeNativeCamera),
     },
   ],
+  [
+    UIConfigContext.Provider,
+    {
+      value: {
+        notReadySectionEnabled:
+          String(appRoot.getAttribute('data-ui-not-ready-section-enabled')) === 'true',
```

Comment:
Can we add this to the `appRoot.dataset` destructuring on line 80 above for consistency with other data properties references?

Reply:
@aduth sure.